nsds_lab_to_nwb/metadata/exp_note_reader.py

```diff
@@ -82,38 +132,35 @@ def read_csvs(self):
                                names=['a', 'values', 'b', 'c'],
                                index_col=1,
                                dtype=type('hello'))
-        raw_meta = raw_meta.iloc[:, 1]
-        raw_meta.dropna(inplace=True)
-        self.meta_df = raw_meta
         
-        block_df = pd.read_csv(block_path_file,
+        
+        raw_block = pd.read_csv(block_path_file,
                                      delimiter=',',
                                      header=2,
                                      dtype=type('hello'))   
-        
-        #clean up block_df
-        for idx, row in block_df.iterrows():
-            try:
-                _ = int(row['block_id'])
-            except:
-                max_row = idx
-                break
-            
-        for column in block_df.columns:
-            if column.startswith('Unnamed'):
-                block_df.drop(column, axis=1, inplace=True)
-        block_df = block_df[:max_row]
-        block_df.dropna(axis=1, how='all', inplace=True)
-        self.block_df = block_df
-         
+        self._raw_meta = raw_meta
+        self._raw_block = raw_block
+    
+    def read_ods(self):
+        """Read ods
+        """
+        path_file = os.path.join(self.path, self.file[0])
+        raw_meta = pd.read_excel(path_file, sheet_name='MetaData',                                  
+                               delimiter=',',
```

Comment:
This should work with newer version of `pandas` if you remove `delimiter`. I don't think `read_excel()` ever had `delimiter` as an argument, but old versions did accept `**kwds`, so it just never caused problems (but isn't being used).

https://pandas.pydata.org/pandas-docs/version/0.25/reference/api/pandas.read_excel.html?highlight=read_excel#pandas.read_excel

Reply:
Cool. That makes sense. I just copy and pasted the arguments from read_csv but it would make sense that there would be no delimiter arg for read excel